frog/imports/client/Wiki/WikiContentComp.js

```diff
@@ -146,7 +145,9 @@ class WikiContentComp extends React.Component<> {
             style={docModeButtonStyle}
             color="primary"
             onClick={() => {
-              this.setState({ docMode: 'edit' });
+              this.props.checkEdit().then(x => {
```

Comment:
can you use a more descriptive variable than x?

Reply:
Thanks for pointing this out!